packages/cf-builder-card/src/CardBuilder.js

```diff
@@ -48,21 +66,7 @@ class CardBuilder extends React.Component {
   }
 }
 
-CardBuilder.propTypes = {
```

Comment:
I'm guessing Flow has trouble type checking props if we attach prop types on the class like this?